### DISCUSSION TITLE
19489d7d-2969-4386-9319-22c4eb245c69

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#17] [General]
# 19489d7d-2969-4386-9319-22c4eb245c69

RDBMS와 NoSQL

https://choo.oopy.io/19489d7d-2969-4386-9319-22c4eb245c69

**Comment by andole98** [comment]
rdb vs noSQL에서 database cap theorem도 찾아보시면 좋을것 같아요
